client/src/app/site/users/models/view-user.ts

```diff
@@ -150,6 +149,26 @@ export class ViewUser extends BaseProjectableViewModel<User> implements Searchab
     public vote_delegations_from_ids(meetingId?: Id): Id[] {
         return this.user.vote_delegations_from_ids(meetingId || this.getEnsuredActiveMeetingId());
     }
+
+    public vote_weight(meetingId?: Id): number {
+        return this.user.vote_weight(meetingId || this.getEnsuredActiveMeetingId());
+    }
+
+    public number(meetingId?: Id): string {
+        return this.user.number(meetingId || this.getEnsuredActiveMeetingId());
```

Comment:
I think you may not have to use `this.getEnsuredActiveMeetingId` here (it throws an error if I remember correctly) -> If there is not active meeting, you can fall back to the default.